promql/promqltest/testdata/subquery.test

```diff
@@ -49,16 +49,16 @@ load 10s
   metric3 0+3x1000
 
 eval instant at 1000s sum_over_time(metric1[30s:10s])
-  {} 394
+  {} 297
 
 # This is (394*2 - 100), because other than the last 100 at 1000s,
 # everything else is repeated with the 5s step.
 eval instant at 1000s sum_over_time(metric1[30s:5s])
-  {} 688
+  {} 591
```

Comment:
While I think it's fine in this case to leave the expression unchanged (and change the expected result, as you did), you need to update the explanation in lines 54 to 56. It's just not true anymore with the new range definition.